src/templates/ContestLayout.tsx

```diff
@@ -193,6 +193,19 @@ const ContestLayout = ({ data }) => {
                 <div>
                   <h1 className="type__headline__xs">{title}</h1>
                   <p>{details}</p>
+                  {contestid === 231 && (
```

Comment:
Should we *in the future* handle this in the database to avoid hardcoding the contestId ?

Reply:
@Simon-Busch yes! But I believe we can leave this hardcoded for now as this is the first qualifier. Let's make sure to clean this up afterward.